src/components/Section.js

```diff
@@ -1,17 +1,15 @@
 export default class Section {
   constructor({ items, renderer }, containerSelector) {
-    this._renderer = renderer;
     this._items = items;
-    this._containerSelector = document.querySelector(`${containerSelector}`);
+    this._renderer = renderer;
+    this._containerSelector = document.querySelector(containerSelector);

```

Comment:
Needs correction: selector is a string, but here we have an element, use `container` as the name: ` this._container = document.querySelector(containerSelector);`